readme.md

```diff
@@ -85,22 +92,30 @@ msbuild openrct2.proj /t:g2
 msbuild openrct2.proj /t:PublishPortable
 ```
 
-### Mac:
-We support native builds for macOS (limited to i386 only for now).
-Make sure that you have [Homebrew](http://brew.sh/) installed and than run the following commands to install all the needed libraries and build OpenRCT2.
-```
-# Install libraries
-./install.sh
+### macOS:
+
+#### Xcode:
 
-# Build OpenRCT2
-./build.sh
+The reccomended way of building OpenRCT2 for macOS is with Xcode. Open the project file OpenRCT2.xcodeproj in Xcode and build from there. Building this way will handle the dependencies for you automatically. You can also invoke an Xcode build from the command line using `xcodebuild`.
 
-# Run the game
-./openrct2
+#### CMake:
+
+A command line version of OpenRCT2 can be built using CMake. This type of build requires you to provide the dependencies yourself. The supported method of doing this is with [Homebrew](http://brew.sh). Once you have Homebrew installed, you can download all the required libraries with this command:
+```
+brew install cmake openssl jansson libpng sdl2 sdl2_ttf speex
+```
+
+Once you have the dependencies installed, you can build the project using CMake using the following commands:
+```
+mkdir build && cd build
```

Comment:
There isn't really a big reason; I copied this from the last time I used CMake. I personally use `&&` for this because if (for some reason) the mkdir command fails, I don't want to try to cd into a directory that may not be there. I will change this back.